packages/node/test/unit/dev.test.ts

```diff
@@ -307,3 +307,43 @@ test('allow setting multiple cookies with same name', async () => {
     child.kill(9);
   }
 });
+
+test('dev server should remove transfer encoding header', async () => {
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (blocking):** check number of assertions, just in case‏

```suggestion
test('dev server should remove transfer encoding header', async () => {
  expect.assertions(2);
```